docs/api.md

```diff
@@ -0,0 +1,34 @@
+# ssh_scan API documentation
+
+**Last Updated:** 2016-09-19 <br>
+**Author:** agaurav77@yahoo.com <br>
+**API Version:** 1
```

Comment:
Can API version be 1.0?  I say this because we might want incremental improvements without major version bumps.